mfr/ext/tabular/render.py

```diff
@@ -35,7 +36,10 @@ def render_html(fp, src=None):
     :return: RenderResult object containing html and assets
     """
 
-    columns, rows = populate_data(fp)
+    try:
+        columns, rows = populate_data(fp)
+    except KeyError:
```

Comment:
Good call. The key in question tends to be a hex value (e.g. `KeyError: '\x94'`), and while in testing it I've only found KeyErrors raised during the rendering of .dta files, I don't know for sure that it's limited to them. `populate_data` seems the safer place to put it if we want it to cover all tabular filetypes without all the duplication.